tox.ini

```diff
@@ -33,7 +33,7 @@ deps =
     boto3>=1.11.0
     pyhive[presto]==0.6.1
     s3transfer>=0.3,<4.0
-    sqlalchemy<1.4
+    sqlalchemy>=2.0,<2.1
```

Comment:
Note that we cannot make this upgrade to sqlachemy 2.x without removing luigi support for Python 3.6 (see Github Actions results).

Reply:
Yeah saw that in the tox, think i might have a workaround will try it out in the morning

Reply:
Ok fix is a polyfill? to support both sqlalchemy 1 & 2 
 I'm not sure what to do about the codecov